### DISCUSSION TITLE
How about a muon detector variant?

### DISCUSSION THREAD

**Discussion by SteveF161** [ozel/DIY_particle_detector#36] [Ideas]
I was looking at this (see link below) as a possible home project but the need to machine cut and polish the plastic scintillator is more than most citizen scientists can cope with. How feasible would it be to extend the alpha/electron design to use a commercially available scintillator and work as a muon detector?

http://www.cosmicwatch.lns.mit.edu/faq

**Comment by ozel** [comment]
the BPW34 diode is quite light-sensitive, but it is still far away from regular photomultiplier tubes or silicon photomultipliers which are close to detecting single photons easily. The latter SiPMs got quite cheap anyway in recent years (20-30 EUR per diode) as you can see in the cosmic watch project. 

By the way, a similar project is https://muonpi.org/. They are actively looking for participants to host cosmic ray detectors connected to Raspberry Pis 24/7 at their place of residence. Just be sure to ask for an assembled detector unit, because otherwise tiny SMD soldering is needed (you could learn that, too, of course :). It's as well an open hardware project.

In the past, I have been cooking a different version of the DIY particle detector that can detect muons directly without a scintillator. Hoping to release more about that one soon.
If you look closely at the current circuit board design, you may note the greek letter Mu :-).
